resources/plugins/QuantME-CamundaPlugin/adaptation/AdaptationPlugin.js

```diff
@@ -219,14 +216,29 @@ export default class AdaptationPlugin extends PureComponent {
     this.setState({ rewriteOpen: false });
   }
 
+
+  async createHybridLoopStuff() {
```

Comment:
Not really related to the functionality of this PR, but could that be renamed to something more useful than stuff? ;)